tests/suites/test_suite_x509parse.function

```diff
@@ -803,14 +803,14 @@ void mbedtls_x509_dn_get_next( char * name_str, int next_merged, char * expected
     out_size = strlen( expected_oids ) + 2;
     ASSERT_ALLOC( out, out_size );
 
-    TEST_ASSERT( mbedtls_x509_string_to_names( &names, name_str ) == 0 );
+    TEST_EQUAL( mbedtls_x509_string_to_names( &names, name_str ), 0 );
 
     ret = mbedtls_x509_write_names( &c, buf, names );
-    TEST_ASSERT( ret > 0 );
+    TEST_LE_S( 0, ret );
```

Comment:
Note: this line changed from `ret > 0` to `0 <= ret` which is not the same in general, but in this particular case this works just as well: we want to ensure we got no error, and all error codes are `< 0`. We don't really care if the function wrote a non-zero number of bytes (which in practice it will, on success). (So, no change requested.)